invokeai/backend/model_management/lora.py

```diff
@@ -123,8 +121,8 @@ def __init__(
 
     def get_weight(self):
         if self.mid is not None:
-            up = self.up.reshape(up.shape[0], up.shape[1])
-            down = self.down.reshape(up.shape[0], up.shape[1])
+            up = self.up.reshape(self.up.shape[0], self.up.shape[1])
+            down = self.down.reshape(self.down.shape[0], self.down.shape[1])
             weight = torch.einsum("m n w h, i m, n j -> i j w h", self.mid, up, down)
```

Comment:
These lines were crashing because `up` was used before it was defined. These changes made the loras seem to work, but please check that it is the right fix.